src/components/buttons/HdLoaderButton.vue

```diff
@@ -182,10 +231,26 @@ export default {
     },
     clicked() {
       this.$emit('click');
-      if (this.state === 'idle') {
-        this.state = 'loading';
-        this.transitionsQue.push(this.fakeLoading);
+
+      if (this.disabled) {
+        return;
+      }
```

Comment:
Just wanna know if the idea of having it.  We want to keep the button's state but we want to make it disabled?

What I mean is, if we set the `<button>` to disabled, we would have the following state:
![image](https://user-images.githubusercontent.com/1734873/77915294-824e0200-7297-11ea-8dc3-bb344934e827.png)

With this `disabled` prop it looks clickable but it doesn't do anything:
![image](https://user-images.githubusercontent.com/1734873/77915339-972a9580-7297-11ea-9fdc-3565ae437cbd.png)

Is it the desired behavior?

Reply:
Yes at the moment if it is disabled it'll be clickable but it won't start loading which is not UX friendly so I'll adjust it.

Reply:
I mean, you migrated those components from MyHomeday, right? Does it make sense this use case there?

Reply:
Yes, I did but so far I never used this component personally.
According to its usage in **myHomeday** this is the desired behavior.